analyzers/src/SonarAnalyzer.CSharp/Facade/CSharpSyntaxKindFacade.cs

```diff
@@ -29,6 +31,7 @@ internal sealed class CSharpSyntaxKindFacade : ISyntaxKindFacade<SyntaxKind>
         SyntaxKindEx.RecordClassDeclaration,
     };
     public SyntaxKind ClassDeclaration => SyntaxKind.ClassDeclaration;
```

Comment:
nitpick: now `ClassDeclaration` is between `ClassAndRecordDeclaration` and `ClassAndModuleDeclarations`, which for me makes it a bit harder to read

Could you move `ClassDeclaration` above `ClassAndRecordDeclaration`

Reply:
note: by "(also in interface etc)" I meant in all files where this applies